gnucash/gnome/gnucash.appdata.xml.in.in

```diff
@@ -19,7 +19,7 @@
       <li>Keep your small business' accounting up to date</li>
       <li>Create accurate reports and graphs from your financial data</li>
       <li>Set up scheduled transactions to avoid repeated data entry</li>
-      <li>Exchange by CSV/FinTS(former HBCI) or import SWIFT-MT9xx/QIF/OFX data including Transaction Matching</li>
+      <li>Exchange by CSV/FinTS or import SWIFT-MT9xx/QIF/OFX data including Transaction Matching</li>
```

Comment:
This needs a more thorough rewrite. GnuCash 5 supports a much wider range of file formats thanks to better AQBanking integration. "CSV/FinTS" doesn't really make sense either, CSV is an import file format while FinTS is a bidirectional network protocol with capablilities beyond importing transactions.

Reply:
CSV and FinTS allow also export, the others only import.
But as that is the feature list of a desktop file, better slit it in 2 list elements Online Banking and File Exchange with distinctions.

Reply:
The FinTS protocol might support exports, but if it does GnuCash doesn't implement it. Actions>Online provides downloading transactions and balances and initiating SEPA transactions.

Yes, GnuCash can export data via CSV, but that has nothing to do with Online Banking.

Reply:
With `initiating SEPA transactions` you "export" transactions to your bank as orders.
AFAIK implemented, but currently not available in Gnucash are only direct debit orders.
Before the port to SEPA we had them, too.

Reply:
The fact that you wrapped export in scare quotes makes my point: It's stretching the meaning of export.